modules/ROOT/pages/security/ssl-framework.adoc

```diff
@@ -100,6 +100,24 @@ If the same certificates are used across all instances of the cluster, make sure
 Multi-host and wildcard certificates are also supported.
 ====
 
+[TIP]
+====
+If setting up intra-cluster encryption as part of a cluster configuration, do ensure that the certificates used on the cluster endpoint supports both Server and Client usage.
+This is because when connecting between the Neo4j servers for clustering, each server will use its own certificate to authenticate as a client on the connection to another server.
```

Comment:
```suggestion
This is because when connecting between the Neo4j servers for clustering, each server uses its own certificate to authenticate as a client on the connection to another server.
```